POWR-50 Bureau/office group type gets more fields

jookyg opened the pull request:
Josh, this is ready for your review. I had to resolve a couple conflicts first, but it seems okay now. Dev site here: https://powr-50-portlandor.pantheonsite.io

joshuami commented:
Just a quick note about this PR, the merge from POWR-49 seems like cruft created by not start from the master branch with a fresh `git pull origin` and then branching from that clean start. Looks like you figured out how to clean it up.

Also, PRs need to have the POWR-[ID] format for Jira to associate them correctly with the story. I fixed the PR title and added some description so that it explained what the story was a bit more.

jookyg commented:
Yeah, I accidentally started 49 from the wrong branch and learned about rebase to fix it.

Understood on the PR titles. I was up half the night coughing, so I don’t fully have my wits about me, trying to make more progress than mistakes today ☺

From: Joshua Mitchell [mailto:notifications@github.com]
Sent: Friday, April 13, 2018 11:55 AM
To: eGovPDX/portlandor <portlandor@noreply.github.com>
Cc: Clapp, Greg <Gregory.Clapp@portlandoregon.gov>; Author <author@noreply.github.com>
Subject: Re: [eGovPDX/portlandor] POWR-50 Bureau/office group type gets more fields (#22)


Just a quick note about this PR, the merge from POWR-49 seems like cruft created by not start from the master branch with a fresh git pull origin and then branching from that clean start. Looks like you figured out how to clean it up.

Also, PRs need to have the POWR-[ID] format for Jira to associate them correctly with the story. I fixed the PR title and added some description so that it explained what the story was a bit more.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/eGovPDX/portlandor/pull/22#issuecomment-381230122>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ASsBBc3vc6xm6xhqbABgK_npPtlThX-fks5toPRmgaJpZM4TT1dO>.
